pages/stylesheets/gov-uk-overrides.scss

```diff
@@ -9,3 +9,14 @@
 #global-cookie-message {
   background-color: #f3f2f1;
 }
+
+.linkStyle {
```

Comment:
is this a gov-uk style?

Reply:
No, but there wasn't something similar in the gov-uk styles to make it look like in the designs

Reply:
Then it's not a gov-uk override. Neither is #global-cookie-message, hadn't noticed that before.